Yoast/javascript
# P3 114 match the keyphrase input errors

## Summary


This PR can be summarized in the following changelog entry:

* Changes the way the error messages of the modal opening button are displayed. Changes indentation and color.

## Relevant technical choices:

* Removed an unused method which @enricobattocchi suggested.
* Added an extra color to colors.json which is used for the error message to display the correct color which the design suggests.

## Test instructions

This PR can be tested by following these steps:

* Checkout free on feature/semrush, build and connect the monorepo (checked out on this branch)
* Go to a post and leave the keyphrase empty and hit the button to open the modal
* Note that an error message should appear which looks like the following design:
![image](https://user-images.githubusercontent.com/9864796/90256308-d53ee500-de45-11ea-85fc-ba841d827620.png)
* Now add a too long keyphrase with multiple comma's and/or spaces
* Make sure that 2 error messages appear which also look like the design
* Try to break the design somehow and make sure it won't break 😃 

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  * N/A

## UI changes

## Quality assurance
